### DISCUSSION TITLE
Keyboard adds extra space on iOS

### DISCUSSION THREAD

**Discussion by henrik-funmed** [flyerhq/react-native-chat-ui#76] [Q&A] [answered]
Hi!

I am using react-navigation/bottom-tabs and the chat is in one of the tabs. When opening the keyboard on iOS extra space is added with, what seems to be, the same height as the bottom tabs. On Android the bottom tabs are displayed there, but that is not how iOS should behave. My chat screen does not make use of SafeAreaView or anything like that. Do you know of a way to get rid of the extra space?

<img width="379" alt="Skärmavbild 2022-12-22 kl  10 22 46" src="https://user-images.githubusercontent.com/83297817/209101725-19110a0f-133d-46a2-a94e-99ed48b86c1b.png">

**Comment by demchenkoalex** [comment, marked as answer]
Hey! Chat wasn't supposed to be used on a tab bar itself + I am not actively maintaining this right now, it comes from a safe area. Better to fork it and change manually.

**Reply by henrik-funmed** [reply]
Ok, thank you.
